molecule/cloud-aws-direct/playbook.yml

```diff
@@ -3,44 +3,173 @@
-      artifact_bucket_name: 's3_artifact_test'
-      jenkins2_ec2_enable: true
-      jenkins2_jira_enabled: true
-      jenkins2_globaltools_enable: true
-      jenkins2_globaltools_nodejs_enable: false
-      jenkins2_globaltools_jdk_version:
-        - jdk-9.0.4-oth-JPR
-      jenkins2_globaltools_maven_version:
-        - 3.6.1
-      jenkins2_globaltools_ant_version:
-        - 1.10.6
-      jenkins2_globaltools_sonarscanner_version:
-        - 3.3.0.1492
-      jenkins2_globaltools_sonarscannermsbuild_version:
-        - 4.6.2.2108
-      jenkins2_globaltools_gradle_version:
-        - 5.5-rc-2
-      jenkins2_globaltools_docker_version:
-        - latest
+      jenkins2_configure_by_casc: true
+      jenkins2_configure_by_groovy: false
+      jenkins2_jcasc_config:
+        credentials:
+          system:
+            domainCredentials:
+            - credentials:
+              - basicSSHUserPrivateKey:
+                  description: "credentials for ubuntu user to connect remotely"
+                  id: "sshconnection"
+                  privateKeySource:
+                    directEntry:
+                      privateKey: "-----BEGIN RSA PRIVATE KEY-----
+
+        MIIEogIBAAKCAQEAn0wfetr65/gxuVloiS0B1GckoKZMwsGdOw48SqtqlWBSG5gN
+        EzR6R9sZI/Wy2KnuDbIUtcQYVHPoZvuL/1qBh6jJmBsOYNJR3gAz6QSWHA7k9s/f
+        vnu0tQJX9+Krf7mB4+fcwTebKbbekSt6zeXKG08fJMqo/sKd9AMhlQeHuidV9aQ7
+        wPeBk9UB0d1CAD/UGAjNmeaS+Y4rmB6hFTanE/MZb1AEite5YetcfDWhA5YZ2Cw8
+        P3ZJLNAUTF9f+eP2uXmQ1dpwiUj59ZHNrsuCPu0uoii574LdlllqRf/j57Wm4PPf
+        MYFp54EPnqwL10XqKbYPXxeRFI8xLQVIzhT+gwIDAQABAoIBADViYm5espVIxQkz
+        oGfQUH
+        qZbjIgMxzNi/ri6W0CxZWd6wPDh21pNXcrM07XAQrQONUmLVZbx9/nPw+mWRG6Q0
+        QyM43dzd8BB0wtqa9RKMIe3Loe+jxxC0P9v4VGEw0qNuCKIQHOc=
+
+        -----END RSA PRIVATE KEY-----"
+                  scope: GLOBAL
+                  username: "ubuntu"
+              - usernamePassword:
+                  description: "username and password for pipeline libraries"
+                  id: "pipeline_libraries"
+                  password: "somepass"
+                  scope: GLOBAL
+                  username: "pipelineLibrariesUser"
+              - usernamePassword:
+                  description: "for operations in AWS"
+                  id: "AWS_CREDENTIALS"
+                  password: "somepass"
+                  scope: GLOBAL
+                  username: "AWSaccessKey"
+              - aws:
+                  accessKey: 'somekey'
+                  description: "for ec2 plugin to create ec2 for slave instances"
+                  id: "AWS_EC2_CREDS"
+                  scope: GLOBAL
+                  secretKey: 'somepasskey'
+              - basicSSHUserPrivateKey:
+                  description: "credentials for hybris user to connect remotely"
+                  id: "sshagent"
+                  privateKeySource:
+                    directEntry:
+                      privateKey: "-----BEGIN RSA PRIVATE KEY-----
+
```

Comment:
please give explanation for this private key

Reply:
This key was placed here as an example, nevertheless,  I have changed all entries to 
"-----BEGIN RSA PRIVATE KEY-----
Here could be placed your private key,
but we strongly recommend keeping private keys in vault service,
like Ansible-vault or Hashicorp-vault, e.t.c
-----END RSA PRIVATE KEY-----"
in the last commit.